src/api.js

```diff
@@ -1,7 +1,7 @@
 const axios = require("axios");
 
-export const fetchLists = (endpoint, params) => {
-  const key = endpoint.split("/").slice(-1)[0];
+export const fetchLists = (endpoint, params, key) => {
```

Comment:
I see what you've done here, and I appreciate that you're aiming to keep your code as dry as possible, but I would argue that in the spirit of "separating out our concerns" that each endpoint should have it's own axios call. (Note - that's each endpoint, not each endpoint AND queries). At the moment it's a bit fiddly to read!

Reply:
I get what you're saying but if I refactored into a function for each endpoint I would find it much harder to read than the current code, I did it like that in the last sprint and found myself constantly getting confused as to what I was using.